extra/mediawiki/api/authors.txt

```diff
@@ -0,0 +1 @@
+Giftpflanze
```

Comment:
Would you prefer to be a named author or a nicknamed author? Either is fine.

If a nickname, maybe use ``@giftpflanze``?

Reply:
I didn't want to use my real name because this would mean deanonymizing my wikipedia username. I think I will still go by Giftpflanze.